docs/components/movement-sensor/_index.md

```diff
@@ -0,0 +1,672 @@
+---
+title: "Movement Sensor Component"
+linkTitle: "Movement Sensor"
+weight: 70
+type: "docs"
+description: "A sensor that measures location, kinematic data, or both."
+tags: ["movement sensor", "gps", "imu", "sensor", "components"]
+icon: "/components/img/components/imu.svg"
+no_list: true
+aliases:
+    - /components/movement-sensor/
+# SME: Rand
+---
+
+A movement sensor component is a sensor that gives data on where a robot is and how fast it is moving.
+Examples of movement sensors include global positioning systems (GPS), inertial measurement units (IMUs), accelerometers and gyroscopes.
+
+{{% alert title="Note" color="note" %}}
+
+Viam also supports generic [sensors](/components/sensor/) and [encoders](/components/encoder/).
+
+{{% /alert %}}
+
+## Configuration
+
+Viam supports several different models of GPS, IMU and accelerometer.
+Click the model names below for configuration information:
+
+Model | Supported hardware <a name="model-table"></a>

```

Comment:
this has turned into a mixture of description (for the rtk station) and supported hardware. Let's make that more consistent

Reply:
RTK station and camera mono do not represent hardware, but I like the "supported hardware" heading instead of "description" because people looking for supported hardware on the doc will see it. Do you think it should change to "Description"? Or should we add something like "Not hardware;" to the non-hardware models?

Reply:
But not all of them describe hardware. If it says a model that derives from... I doesn't really fit?

Reply:
Let's leave it for now and look at it when we make sure the components pages are consistent

Reply:
Yeah I totally see your point. Revisiting sounds good.